### ISSUE TITLE
The screen reader does not announce the updated search results when we search in the search box: A11y_ML.NET Model Builder_Column Settings_Screenreader.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/machinelearning-modelbuilder#3023]
### GitHubTags
#A11yUsable; #A11yTCS;#DesktopApp; #Win32;#Model Builder - Machine Learning for .NET Developers;#BM-MlnetModelBuilder-Win32-March25;#Win32;

### Actual Result: 
The screen reader does not announce the number of results when we search in the search box; it only announces that no columns were found.
### Expected Result:
The screen reader should announce the number of results found when searching in the search box, for example: "2 results found," "3 results found".

### Environment Details:
Application Name: VS Setup
Edition Windows 11 Enterprise
Version 24H2
OS build 26100.3476
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.13.3
### Repro Steps:
**Prerequisites:**
Install latest Visual Studio 2022 with at least one of the .NET workloads (ASP.NET and web development, .NET desktop development, or .NET Core cross-platform development) and/or the ML.NET Model Builder component selected.

1. Launch visual studio.
2. Turn on NVDA/JAWS.
3. Create a new .NET project or open an existing .NET project.
4. Right-click on the project in the solution explorer and Add > Machine Learning.
5. Tab till scenario and select a scenario.
6. Select training environment (Select local(CPU).
7. Tab till next step and press enter, "select file from the data source type and add a file".
8. Tab till advanced data options and press enter.
9. Tab till Column settings and press enter.
10. Verify that Screen reader should announce the how many results are coming when search in the search box for ex: 2 results found, 3 results found.

### User Impact:
Visually impaired users who rely on screen readers for navigation will not be aware of the number of search results shown, which can lead to confusion and difficulty in navigating the content.

### Attachment:

https://github.com/user-attachments/assets/4fb32456-fdc8-485f-bfb0-3a34cf108a15

JAWS

https://github.com/user-attachments/assets/8d565683-8690-4292-8660-b4749054b578

**Comment by Mana-swi** [comment]
#a11ydb-search-count-silent;
